sources/il/countrywide-hebrew.json

```diff
@@ -0,0 +1,16 @@
+{
+    "coverage": {
+        "country": "il",
+        "ISO 3166": {"alpha2": "IL", "country": "Israel"}
+    },
+    "data": "http://www.sol-israel.co.il/arcgis/rest/services/Amidar_Vector/MapServer/9",
+    "type": "ESRI",
+    "language": "IW",
+    "conform": {
+        "type": "geojson",
+        "city": "SETL_NAME",
+        "number": "HOUSE_NUM",
+        "unit": "ENTRY_LETR",
```

Comment:
Mentioned this in the PR,  but there was other licensing discussion going on as well. Entrance letter is part of the house number, not the unit. The "format" conform function can be used to concatenate multiple fields without space:

```json
"number": {
    "function": "format",
    "fields": ["HOUSE_NUM", "ENTRY_LETR"],
    "pattern": "$1$2"
}
```